ui/src/app/machines/views/MachineDetails/MachineLogs/DownloadMenu/DownloadMenu.tsx

```diff
@@ -16,9 +22,21 @@ export const DownloadMenu = (): JSX.Element => {
           {
             children: "curtin-logs.tar",
           },
-          {
-            children: "Installation output",
-          },
+          ...(installationOutput.log
+            ? [
+                {
+                  children: "Installation output",
+                  onClick: () => {
+                    if (installationOutput.log) {
+                      fileDownload(
+                        installationOutput.log,
+                        "installation-output.txt"
```

Comment:
Could we help the user out by naming the file after the machine or something?

Reply:
Maybe even a date on it?

Reply:
Good idea!